nodes/pods/nodes-pods-secrets-store.adoc

```diff
@@ -0,0 +1,47 @@
+:_content-type: ASSEMBLY
+[id="nodes-pods-secrets-store"]
+= Providing sensitive data to pods by using an external secrets store
+include::_attributes/common-attributes.adoc[]
+:context: nodes-pods-secrets-store
+
+toc::[]
+
+// Pulled from the other assembly. Keep?
+Some applications need sensitive information, such as passwords and user names, that you do not want developers to have.
+
+As an alternative to using Kubernetes `Secret` objects to provide sensitive information, you can use an external secrets store to store the sensitive information. You can use the {secrets-store-operator} to integrate with an external secrets store and mount the secret content as a pod volume.
+
+// TODO: Might not need this if it's in the module below (or vice versa)
+:FeatureName: The Secrets Store CSI Driver Operator
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+// About the {secrets-store-operator}
+include::modules/persistent-storage-csi-secrets-store-driver-overview.adoc[leveloffset=+1]
+
+// Available secrets store providers
+include::modules/secrets-store-providers.adoc[leveloffset=+2]
+
+// Automatic rotation
+include::modules/secrets-store-auto-rotation.adoc[leveloffset=+2]
+
+== Installing the Secrets Store CSI Driver
+
+TODO: Include Lisa's module
+
+// Installing the Secrets Store CSI Driver
+
+// Mounting secrets from an external secrets store to a CSI volume
+include::modules/secrets-store-azure.adoc[leveloffset=+1]
+
+// Enabling synchronization of mounted content as Kubernetes secrets
+include::modules/secrets-store-sync-secrets.adoc[leveloffset=+1]
+
+// Viewing the status of secrets in the pod volume mount
+include::modules/secrets-store-viewing-secret-versions.adoc[leveloffset=+1]
+
+// TODO: Do we need to talk about metrics? Do users need to do anything to collect metrics?
```

Comment:
@ropatil010 @knelasevero

Do we need to make any mentions about metrics for this? Is it done automatically, or is there anything users need to do to enable metrics collection?